config/env_spec_request.go

```diff
@@ -324,7 +430,7 @@ func (e *EnvironmentSpecRequest) IsAuthorizationRequired() bool {
 	return !e.GetConsumerAuthorization().Disabled && !e.GetConsumerAuthorization().isEmpty()
 }
 
-func (e *EnvironmentSpecRequest) GetHTTPRequestTransformations() (transforms HTTPRequestTransformations) {
+func (e *EnvironmentSpecRequest) GetHTTPRequestTransformations() (transforms HTTPRequestTransforms) {
```

Comment:
nit: should we change the function name to `...Transforms`?

Reply:
done